save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/docker/ContainerManager.kt

```diff
@@ -148,13 +156,20 @@ class ContainerManager(private val settings: DockerSettings) {
         val hostIp = getHostIp("host.docker.internal")
         log.debug("Resolved host IP as $hostIp, will add it to the container")
         val buildImageResultCallback: BuildImageResultCallback = try {
-            dockerClient.buildImageCmd(dockerFile)
+            val buildCmd = dockerClient.buildImageCmd(dockerFile)
                 .withBaseDirectory(tmpDir)
                 .withTags(setOf(imageName))
                 .withExtraHosts(hostIp?.let {
                     setOf("host.docker.internal:$hostIp")
                 } ?: emptySet())
-                .start()
+            buildCmd.execTimed(meterRegistry, "save.orchestrator.docker.build", "baseImage", baseImage) { record ->
```

Comment:
What if `buildCmd` will be an `emptySet`, which behavior we will see? I don't see any `isEmpty` like checks

Reply:
No, `emptySet` is a possible value for `extraHosts` parameter. `buildCmd` is always a docker command instance

Reply:
Ok, didn't notice